sklearn/feature_extraction/tests/test_image.py

```diff
@@ -13,7 +14,10 @@
     img_to_graph, grid_to_graph, extract_patches_2d,
     reconstruct_from_patches_2d, PatchExtractor, extract_patches)
 from sklearn.utils.graph import connected_components
+from sklearn.utils.testing import SkipTest
 
+if sp.__version__ < '0.12.0':
+    raise SkipTest("Skipping because SciPy version earlier than 0.12.0 and thus does not include the misc.face() image.")
```

Comment:
Please insert a line break to respect pep8's 80  columns limit.

```python
    raise SkipTest("Skipping because SciPy version earlier than 0.12.0 and "
                   "thus does not include the misc.face() image.")
```